src/cforth/order.fth

```diff
@@ -16,13 +16,13 @@ also root definitions
 : set-order  ( wid1 .. widn n -- )
    context  /context  erase
```

Comment:
I tried this, but it didn't make a difference.  It seems that `token@` treats 0 specially.

However, after re-using the code from `vocabulary` to initialize wordlists (in commit 7f17bde) everything seems to work as it should.